installer.sh

```diff
@@ -63,63 +63,66 @@ get_os_info() {
+
+    # TODO: Consider installing into /usr/local/bin and /usr/local/share instead of into
+    #   ~/.local/share and ~/.local/bin, since those are always on the PATH and are standard
+    #  to install programs like this. But then we need to run some commands below with sudo.
+
+    ##### Download and install the specified wasp release. #####
+
+    DATA_DST_DIR="$HOME_LOCAL_SHARE/wasp-lang/$VERSION_TO_INSTALL"
     create_dir_if_missing "$DATA_DST_DIR"
-    BIN_DST_DIR="$HOME_LOCAL_BIN"
-    create_dir_if_missing "$BIN_DST_DIR"
 
-    # If our install locations are already occupied (by previous wasp installation or smth else),
-    # inform user that they have to clean it up (or if FORCE is set, we do it for them).
+    if [ -z "$(ls -A "$DATA_DST_DIR")" ]; then
+        PACKAGE_URL="https://github.com/wasp-lang/wasp/releases/download/v${VERSION_TO_INSTALL}/${BIN_PACKAGE_NAME}"
+        make_temp_dir
+        info "Downloading binary package to temporary dir and unpacking it there...\n"
+        dl_to_file "$PACKAGE_URL" "$WASP_TEMP_DIR/$BIN_PACKAGE_NAME"
+        echo ""
 
-    OCCUPIED_PATH_ERRORS=""
-    if [ -e "$DATA_DST_DIR/wasp" ]; then
-        if [ "$FORCE" = "true" ]; then
-            info "Removing already existing $DATA_DST_DIR/wasp."
-            rm -r "$DATA_DST_DIR/wasp"
-        else
-            OCCUPIED_PATH_ERRORS=$OCCUPIED_PATH_ERRORS"Directory $DATA_DST_DIR/wasp already exists.\n"
+        info "Installing Wasp data to $DATA_DST_DIR.\n"
+        if ! tar xzf "$WASP_TEMP_DIR/$BIN_PACKAGE_NAME" -C "$DATA_DST_DIR"; then
+            die "Installing data to $DATA_DST_DIR failed: unpacking binary package failed."
         fi
-    fi
-    if [ -e "$BIN_DST_DIR/wasp" ]; then
-        if [ "$FORCE" = "true" ]; then
-            info "Writing over existing $BIN_DST_DIR/wasp."
-        else
-            OCCUPIED_PATH_ERRORS=$OCCUPIED_PATH_ERRORS"Binary file $BIN_DST_DIR/wasp already exists.\n"
-        fi
-    fi
-    if [ ! -z "$OCCUPIED_PATH_ERRORS" ]; then
-        die "\nInstallation failed!\n\n${OCCUPIED_PATH_ERRORS}\nRemove listed entries manually or run the installer with --force flag to write over them:\n\n  ${BOLD}curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s -- --force${RESET}\n"
+    else
+        info "Found already existing installation on the disk, at $DATA_DST_DIR. Using it instead.\n"
     fi
 
-    info "Installing Wasp data to $DATA_DST_DIR/wasp."
-    if ! mv "$WASP_TEMP_DIR/wasp" "$DATA_DST_DIR/"; then
-        die "Installing data to $DATA_DST_DIR failed."
-    fi
+    ##### Create executable that uses installed wasp release. #####
+
+    BIN_DST_DIR="$HOME_LOCAL_BIN"
+    create_dir_if_missing "$BIN_DST_DIR"
 
-    info "Installing Wasp executable to $BIN_DST_DIR/wasp."
+    if [ -e "$BIN_DST_DIR/wasp" ]; then
+        info "Configuring Wasp executable at $BIN_DST_DIR/wasp to use wasp version $VERSION_TO_INSTALL."
+    else
+        info "Installing Wasp executable to $BIN_DST_DIR/wasp."
```

Comment:
Super small, but do we want to use "Wasp" or "wasp" throughout? It is mixed currently.

Reply:
Done, now it is all `wasp`!